ftw/theming/resources/scss/globals/variables.scss

```diff
@@ -198,6 +196,80 @@ $screen-XXS: "(min-width: #{$mobile-tiny})" !default;
   }
 }
 
+$primary-color: #205C90 !default;
+$secondary-color: #75ad0a !default;
+
+$gray-base: #403e3d !default;
+$gray-darker: lighten($gray-base, 13.5%) !default;
+$gray-dark: lighten($gray-base, 20%) !default;
+$gray: lighten($gray-base, 33.5%) !default;
+$gray-light: lighten($gray-base, 46.7%) !default;
+$gray-lighter: lighten($gray-base, 70%) !default;
+
+$standalone-color: #fafafa !default;
+$context-color: #337ab7 !default;
+$success-color: #5cb85c !default;
+$warning-color: #f0ad4e !default;
+$danger-color: #d9534f !default;
+
+$page-bg-color: $gray-lighter !default;
+$content-bg-color: #fff !default;
+$text-color: $gray-dark !default;
+$light-text-color: $gray-light !default;
+$link-color: $primary-color !default;
+$link-font-weight: normal !default;
+$link-hover-color: $secondary-color !default;
+$link-focus-color: $secondary-color !default;
+$link-decoration: none !default;
+$link-hover-decoration: underline !default;
+$outline-color: change-color($primary-color, $lightness: 60%);
```

Comment:
Please also add `!default` to ` $standard-iconset` and `$use-font-awesome-mimetype-icons` (my fault).